examples/jfrog-platform/HA-with-distribution-S3/readme.md

```diff
@@ -0,0 +1,105 @@
+## 3-Node Artifactory Cluster with Distribution and direct-S3 Provider
+
+### Overall
+| Product   | Enabled   |
+|-------------|-------------|
+| Artifactory      | ✅      |
+| xray | ❌ |
+| distribution | ✅ |
+| insight | ❌ |
+| pipelines | ❌ |
+| worker | ❌ |
+
+
+### Artifactory
+| Detail   | Value    |
+|-------------|-------------|
+| Replica      | 3      |
+| Database | External Postgres |
+| Persistence | Default Storage Class + S3 |
+| SSL | ✅ |
+| Ingress | ❌ |
+| Nginx Deployment | ✅ |
+| UnifiedSecret | ✅ |
+| Non-Default Admin Credential | ✅ |
+| Default Master Key |  ❌ |
+| Sizing Parameters | artifactory-xlarge |
+| Private Registry | ✅ |
+
+
+### Distribution
+
+| Detail   | Value    |
+|-------------|-------------|
+| Replica      | 2       |
+| Database | External Postgres |
+| Persistence | Default Storage Class |
+| External redis | ❌ |
+| SSL | ✅ |
+| UnifiedSecret | ❌ |
+| Sizing Parameters | distribution-xlarge |
+| Private Registry | ✅ |
+
+
+Note: This requires distribution chart 102.23.0+ to work, which comes default with platform chart 10.17.4+.
```

Comment:
I would avoid this comment as it's confusing and brings up more questions. Since we already have 10.20.x out, we should be good.

Reply:
Sorry, I should put this under the section with openshift. The original case was on openshift which distribution chart doesn't support until 102.23.0